js/client.js

```diff
@@ -845,7 +846,8 @@
 					name: name,
 					userid: toUserid(name),
 					named: named,
-					avatar: parts[3]
+					avatar: parts[3],
+					isStaff: !!+parts[4]
```

Comment:
Are you actually going to add `isStaff` to userdetails? Ugh. I'd say just make it a preference.